shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/ManagedBridgeServiceImpl.java

```diff
@@ -118,7 +118,8 @@ private void createOrUpdateBridgeSecret(ManagedBridge managedBridge) {
     @Override
     public void deleteManagedBridge(BridgeDTO bridgeDTO) {
 
-        ManagedBridge mb = ManagedBridgeConverter.fromBridgeDTOToManageBridge(bridgeDTO, null);
+        String bridgeNamespace = namespaceProvider.getNamespaceName(bridgeDTO.getId());
+        ManagedBridge mb = ManagedBridgeConverter.fromBridgeDTOToManageBridge(bridgeDTO, bridgeNamespace);
 
```

Comment:
@vaibhavjainwiz You need to delete the `ManagedBridge` instance here if it exists.

Reply:
And could you please add a unit test for this method as well. Thanks!

Reply:
@robpblake `Delete` operator had a inbuild handling for this scenerio. `Delete` would return null or false if `ManagedBridge` is not found. I had added the testcase for this method.